libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/preview/CarRouteLine.kt

```diff
@@ -129,10 +148,9 @@ class CarRouteLine(
 
     private fun onRoutesChanged(routes: List<NavigationRoute>) {
         logAndroidAuto("CarRouteLine onRoutesChanged ${routes.size}")
-        mainCarContext.mapboxCarMap.carMapSurface?.getStyle()?.let { style ->
-            if (routes.isNotEmpty()) {
```

Comment:
We don't perform this check anymore, is it expected?

Reply:
my logic here is wrong, am updating